packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -158,8 +158,11 @@ export const ItemBaseUI = styled.li`
 	margin-bottom: 0;
 
 	button,
+	a.components-button,
 	a {
 		width: 100%;
+		color: ${ G2.lightGray.ui };
+		padding: ${ space( 0.75 ) } ${ space( 2 ) };
```

Comment:
Moving this here would leave simple text items (`isText`) without appropriate padding.

See `Navigation > More Examples` in the Storybook:

<img width="315" alt="Screenshot 2021-02-15 at 13 32 57" src="https://user-images.githubusercontent.com/2070010/107952993-637d5a80-6f92-11eb-876d-e6026a59e9b0.png">

Reply:
>Moving this here would leave simple text items (isText) without appropriate padding.

Good find, thank you. I added this padding back to the `ItemUI`